04 DisplayData/src/api/memberAPI.js

```diff
@@ -0,0 +1,16 @@
+var memberMockData_1 = require('./memberMockData');
```

Comment:
That's an issue when using webstorm, auto create js files out of dist, please disable this option in your ide

Sent from my Windows Phone

-----Mensaje original-----
De: "Nasdan" <notifications@github.com>
Enviado: ‎22/‎03/‎2016 22:35
Para: "Lemoncode/react-typescript-samples" <react-typescript-samples@noreply.github.com>
Asunto: Re: [react-typescript-samples] Code cleanup (#36)

In 04 DisplayData/src/api/memberAPI.js:
> @@ -0,0 +1,16 @@
> +var memberMockData_1 = require('./memberMockData');
ignore .js and .js.map files
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub